test/plug/adapters/test/conn_test.exs

```diff
@@ -114,6 +114,24 @@ defmodule Plug.Adapters.Test.ConnTest do
     assert child_conn.host == "www.elixir-lang.org"
   end
 
+  test "sent chunks" do
+    conn = conn(:get, "/")
+    conn = Plug.Conn.send_chunked(conn, 200)
+    {:ok, conn} = Plug.Conn.chunk(conn, "foo")
+    {:ok, conn} = Plug.Conn.chunk(conn, "bar")
+
+    assert conn.resp_body == "foobar"
+    assert Plug.Test.sent_chunks(conn) == ["foo", "bar"]
+  end
+
+  test "sent chunks on send_resp" do
+    conn = conn(:get, "/")
+    conn = Plug.Conn.send_resp(conn, 200, "foobar")
+
+    assert conn.resp_body == "foobar"
+    assert Plug.Test.sent_chunks(conn) == []
+  end
```

Comment:
When doing `Req.get(plug: plug, into: [])`, I'm using this new sent_chunks/1 function however as seen in this test it doesn't behave correctly (for Req) when the plug is not sending chunks. Maybe the function should raise?